doc-content/enterprise-only/using-ansible-rhdm-in-ONAP/rules-engines-and-rhdm-con.adoc

```diff
@@ -0,0 +1,21 @@
+[id='rules-engines-and-rhdm-con']
+= Rules Engines and {PRODUCT}
+
+A rules engine is a software element that makes decisions based on business logic built using declarative format. Rules engine can be very basic providing support for simple rules to very complicated based on sophisticated algorithms.
+
+{PRODUCT} provides a versatile set of decision capabilities, including a rules engine based on the link:https://access.redhat.com/documentation/en-us/red_hat_jboss_bpm_suite/6.2/html/development_guide/rule_evaluation_with_phreak_algorithm[PHREAK] algorithm (an evolution of the well know Rete algorithm), a link:https://www.omg.org/spec/DMN/About-DMN/[DMN v1.1 FEEL Compliance Level 3] runtime, support for link:https://www.packtpub.com/mapt/book/networking_and_servers/9781783288625/7/ch07lvl1sec47/pmml[PMML], as well as a Complex Event Processing engine.
```

Comment:
We are pointing to the 6.2 version of the dev guide here. We do not support 6.2 version and pointing to the previous version of the docs is not ideal. So please remove those references to the earlier version of the product docs.
Also, regarding the two other references (to https://www.omg.org/spec/DMN/About-DMN/ and https://subscription.packtpub.com/book/networking_and_servers/9781783288625/7/ch07lvl1sec47/pmml), I'd prefer if we take Christy's ack on pointing to these resources from our doc.

Reply:
I have updated the link to point to the 6.4 version of the dev guide which we are supporting now. 

Hi @Christywatkins, can we point to the following resources from this doc:
-  https://www.omg.org/spec/DMN/About-DMN/ and 
- https://subscription.packtpub.com/book/networking_and_servers/9781783288625/7/ch07lvl1sec47/pmml